API_CONTRACTS.md

```
@@ -12,16 +12,19 @@ All endpoints support **100% mock mode** for CI/CD and development without exter
 ## 🔐 Authentication
 
 Most endpoints require authentication via JWT token in `Authorization` header:
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)